osmosis-1/osmosis.zone_assets.json

```diff
@@ -940,6 +940,18 @@
       "path": "transfer/channel-258/mO9W",
       "osmosis_verified": false
     },
+    {
+      "chain_name": "odin",
+      "base_denom": "udoki",
+      "path": "transfer/channel-258/udoki",
+      "osmosis_verified": false
+    },
```

Comment:
Please add "osmosis_unlisted": true 
to both assets.
Also please move this addition to the bottom of the list